docs/code/microsoft-ml-tokenizers-migration-guide.md

```diff
@@ -0,0 +1,19 @@
+# Porting to Microsoft.ML.Tokenizers
+
+This guide provides general guidance on how to migrate from various tokenizer libraries to `Microsoft.ML.Tokenizers` for Tiktoken.

```

Comment:
Do we plan to add others beyond deepdev?

Reply:
Yes. That's why we added the DeepDev guidance as its own section and kept the doc more generic.